dotCMS/src/main/java/com/dotcms/rest/api/v1/workflow/WorkflowResource.java

```diff
@@ -1705,18 +1711,19 @@ private void saveMultipleContentletsByDefaultAction(final List<Map<String, Objec
                 final IndexPolicy indexPolicy = MapToContentletPopulator.recoverIndexPolicy(
                         singleFireActionForm.getContentletFormData(),
                         IndexPolicyProvider.getInstance().forSingleContent(), request);
-                final Contentlet contentlet = this.getContentlet
-                        (inode, identifier, languageId,
-                                ()->WebAPILocator.getLanguageWebAPI().getLanguage(request).getId(),
-                                singleFireActionForm, initDataObject, mode);
+
 
                 try {
 
                     fireTransactionalAction(systemAction, singleFireActionForm, request, mode,
                             initDataObject, resultMap, inode, identifier, languageId, user, indexPolicy);
                 } catch (Exception e) {
 
-                    final String id = UtilMethods.isSet(identifier)?identifier:inode;
+                    final String id = UtilMethods.isSet(identifier) ? identifier :
+                            (UtilMethods.isSet(inode) ? inode:
+                                    null != singleFireActionForm && null != singleFireActionForm.getContentletFormData() ?
+                                            singleFireActionForm.getContentletFormData().toString() :
+                                            "unknown contentlet" + System.currentTimeMillis());
```

Comment:
is not need any curl-test for this change?

Reply:
yup working on it